tools/cli/commands/release.js

```diff
@@ -167,7 +172,7 @@ export async function scriptRouter( argv ) {
 			argv.script = `vendor/bin/changelogger`;
 			argv.scriptArgs = [ `write`, `--amend` ];
```

Comment:
Should probably include logic to handle --add-pr-num here too.

Reply:
Thanks, I missed that one: 620d7e3